src/libs/actions/Report.js

```diff
@@ -421,6 +426,40 @@ function navigateToAndOpenReport(userLogins) {
     Navigation.navigate(ROUTES.getReportRoute(reportID));
 }
 
+/**
+ * This will navigate to an existing thread, or create a new one if necessary
+ *
+ * @param {String} childReportID The reportID we are trying to open
+ * @param {Object} parentReportAction the parent comment of a thread
+ * @param {String} parentReportID The reportID of the parent
+ *
+ */
+function navigateToAndOpenChildReport(childReportID = '0', parentReportAction = {}, parentReportID = '0') {
```

Comment:
I think the order should be childReportID, parentReportID and then parentReportAction.
I don't think any of them should be optional. The front end has values for all three so send all three.

Reply:
oh it's used to determine if we are opening an existing report vs making a new one. Cool this is fine with me.